src/metabase/search/in_place/filter.clj

```diff
@@ -304,9 +304,11 @@
                 verified
                 non-temporal-dim-ids
                 has-temporal-dim
-                display-type]} search-context
+                display-type
+                is-superuser?]} search-context
         feature->supported-models (feature->supported-models)]
     (cond-> models
+      (not   is-superuser?)        (disj "transform")
```

Comment:
Given the in-place engine was not deprecated as aggressively as we anticipated, evolving parallel logic is unfortunate.

While it would be simple enough to depend on the specification here, I guess having only fragments of the specification apply could be really confusing.

I guess this is fine because the the transform api test enforces parity.